.github/workflows/php-cs-fixer.yaml

```diff
@@ -17,19 +17,9 @@ permissions:
     contents: read
 
 jobs:
-    php-cs-fixer:
-        permissions:
-            contents: write  # for stefanzweifel/git-auto-commit-action to push code in repo
-        runs-on: ubuntu-latest
-        steps:
-            - uses: actions/checkout@v4
-              with:
-                  ref: ${{ github.event.pull_request.head.ref }}
-                  repository: ${{ github.event.pull_request.head.repo.full_name }}
-
-            - name: PHP-CS-Fixer
-              uses: docker://oskarstark/php-cs-fixer-ga:latest
-
-            - uses: stefanzweifel/git-auto-commit-action@v5
-              with:
-                  commit_message: Apply php-cs-fixer changes
+    php-style:
+        uses: pimcore/workflows-collection-public/.github/workflows/reusable-php-style.yaml@main
```

Comment:
@robertSt7 Do this workflow also support pull requests? See https://github.com/pimcore/admin-ui-classic-bundle/pull/517

Reply:
@blankse It support pull request from forks(now).